.env.example

```
@@ -86,4 +86,14 @@ TEST_RUN_FRAME_NUMBER=1000
\ No newline at end of file
+VIDEO_DEFAULT_FPS=50
+
+# Caching optimization for huggingface, ollama, etc. 
+# CAUTION: SETUP ASSUMES LUXNIX USER 'admin' WITH HOME /home/admin
+
+HF_HOME=/home/admin/.cache/huggingface
+HF_HUB_CACHE=/home/admin/.cache/huggingface/hub
+HF_HUB_ENABLE_HF_TRANSFER=1
+OLLAMA_KEEP_ALIVE=4h
+OLLAMA_MODELS=/home/admin/.ollama/models
+TRANSFORMERS_CACHE=/home/admin/.cache/huggingface/hub
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)